Editor/CompileCesiumForUnityNative.cs

```diff
@@ -216,6 +221,23 @@ public void OnPostBuildPlayerScriptDLLs(BuildReport report)
                 BuildNativeLibrary(GetLibraryToBuild(report.summary));
             }
         }
+        
+        public void OnPostprocessBuild(BuildReport report)
+        {
+#if UNITY_IOS
+            if(report.summary.platform == BuildTarget.iOS)
+            {
+                string projectPath = report.summary.outputPath + "/Unity-iPhone.xcodeproj/project.pbxproj";
+                PBXProject pbxProject = new PBXProject();
+                pbxProject.ReadFromFile(projectPath);
+                string target = pbxProject.GetUnityMainTargetGuid();
+                pbxProject.SetBuildProperty(target, "ENABLE_BITCODE", "NO");
```

Comment:
Should we be building our native code with bitcode enabled, rather than disabling it here?